tests/framework/db/DatabaseTestCase.php

```diff
@@ -120,7 +120,7 @@ protected function replaceQuotes($sql)
             case 'cubrid':
             case 'pgsql':
             case 'oci':
-                return str_replace(['[[', ']]'], '"', $sql);
+                return str_replace(['\\[', '\\]'], ['[', ']'], preg_replace('/(\[\[)|((?<!(\[))\]\])/', '"', $sql));
```

Comment:
Why changes for pgsql affects cubrid and oci?

Reply:
fixed.